seed/index.js

```diff
@@ -1,12 +1,24 @@
 require("dotenv").config()
 
 const SpoofItem = require("./../api/models/spoofitem")
-const mongoose = require("mongoose")
-mongoose.connect(process.env.MONGODB_URI, {
-  useMongoClient: true
+const {mongoose} = require('./../config/mongoose')
+const Counter = require("./../api/models/counter")
+
+
+//Clear Collections to start new + add counter
+SpoofItem.remove({}).then((result) => {
+  console.log('spoof items cleared')
+})
+
+Counter.remove({}).then((result) => {
+  console.log('counters cleared')
 })
-const db = mongoose.connection
-db.on("error", console.error.bind(console, "MongoDB connection error:"))
+
```

Comment:
Consider using `Promise.all` to wait until all promises have resolved and then populate the database. Here's an example: https://gist.github.com/kyleva/00d5a12414afe0ec46e513b3241e0471